src/mlpack/tests/akfn_test.cpp

```diff
@@ -11,29 +11,27 @@
 #include <mlpack/core.hpp>
 #include <mlpack/methods/neighbor_search/neighbor_search.hpp>
 #include <mlpack/core/tree/cover_tree.hpp>
-#include <boost/test/unit_test.hpp>
-#include "test_tools.hpp"
+#include "test_catch_tools.hpp"
```

Comment:
Pretty sure we can delete the header, since we don't use `CheckMatrix` etc.

Reply:
We are using `REQUIRE_RELATIVE_ERR(distancesApprox(i), distancesExact(i), epsilon);`